Runtime/Scripts/LyumaAv3Runtime.cs

```diff
@@ -1072,7 +1073,27 @@ void Awake()
 				ViewAnimatorOnlyNoParams = this.emulator.DefaultAnimatorToDebug;
 			}
 
-			animator = GetOrAddComponent<Animator>(this.gameObject);
```

Comment:
I don't think the changes in this file are needed (unless you saw otherwise). I must have been mistaken about av3emu not picking up animators changed during preprocessing hooks, since it doesn't access the Animator at all before it invokes them.

Reply:
From what I know the issues happened in NormalUpdate because the animator was recreated after our Awake happened, probably during Start or during your VRCFury initialization. 

Right now I've made these changes to ensure we still work if someone else recreates the animator during start, and the restart is there to make sure we have the correct defaults once we initialize the animators. Does this make sense?